### DISCUSSION TITLE
Simplified pinned discussions

### DISCUSSION THREAD

**Discussion by EricKotato** [community/community#2878] [Discussions]
It would be nice to have something like "simple mode" for pinned conversation.

All current colors are too bright for dark mode, even if you select cool colors:

![](https://user-images.githubusercontent.com/2903496/101586888-ed89cf80-39f3-11eb-824f-931c15a67f7e.png)

Also they take too much height on the page, even though most of it is unused.

Here is my concept of how it could look:

![](https://user-images.githubusercontent.com/2903496/101587577-8e2cbf00-39f5-11eb-8138-cf64f3e31229.png)

**Comment by becca** [comment]
Thank you for this feedback! Are you finding that they take up too much space or that you don't want the added customization/colors? Unclear if you're requesting both or really one takes priority over the other. 

I understand the height request but if you could expand on the design one and why you're requesting a change that would be great.

**Reply by EricKotato** [reply]
I'm requesting both, sorry for being unclear. But I think that color option is a priority.

Height change could be nice, because you can see more new/top discussions on the screen.

But with color option (especially I'm talking about darker color options) you can set a color that is easy on eyes, especially in dark mode.
